data/pigui/modules/radar.lua

```diff
@@ -17,13 +15,29 @@ local pionillium = ui.fonts.pionillium
 local colors = ui.theme.colors
 local icons = ui.theme.icons
 
+local SCREEN_BORDER = 4
+
 local MAX_RADAR_SIZE = 1000000000
 local MIN_RADAR_SIZE = 1000
+local DEFAULT_RADAR_SIZE = 10000
 
-local shouldDisplay2DRadar
-local current_radar_size = 10000
+local shouldDisplay2DRadar = false
 local blobSize = 6.0
 
+local input_group = 'ShipHUD.RadarControl'
+local keys = {
+	radar_reset = Input.RegisterActionBinding('BindRadarZoomReset', input_group, { activator = { key = Input.keys['slash'] } }),
+	radar_toggle_mode = Input.RegisterActionBinding('BindRadarToggleMode', input_group, { activator = { key = Input.keys['semicolon'] } }),
```

Comment:
Ah, I kinda thought it odd that we're using strings instead of symbols. Another reason to have used UPPERCASE ;)

Alternatively, could do `key_return` or `KEY_RETURN`.  Often a good idea to prefix constant symbols..

Feedback: it would be good to provide the intended use-case as a sample, not the exception, when adding a new feature.

Fixed in 1b370e33f984658232158bddafb4e4840b8a9df1

Reply:
Given the overall convention of the codebase is `ui.keys.xxx` I originally thought it was close enough to self-obvious that `Input.keys[...]` would be the outlier - especially given that I didn't intend TestInput to be anything other than a quick validation for the new code.

That being said, I can see where people might want to copy the code as-is - I'll see about addressing that later.

Reply:
Forgive me; I'm still new to the codebase and when I search for examples of how to do something in this case I only found the TestInput section.

Reply:
No forgiveness necessary!